lib/src/component_declaration/component_base_2.dart

```diff
@@ -68,10 +68,6 @@ ReactDartComponentFactoryProxy2 registerComponent2(react.Component2 Function() d
     bridgeFactory: UiComponent2BridgeImpl.bridgeFactory,
   );
 
-  if (displayName != null) {
```

Comment:
Since `displayName` parameter is no longer used, can we deprecate it, and update the builder to no longer pass it in the generated `registerComponent2` call?

Reply:
Indeed! I was planning on leaving `registerComponent` alone since that whole stack is deprecated, but I ended up updating that too to avoid unnecessary conditional logic in the builder!